logfire/_internal/config.py

```diff
@@ -207,14 +208,22 @@ class CodeSource:
     revision: str
     """The git revision of the code e.g. branch name, commit hash, tag name etc."""
 
-    root_path: str
+    root_path: str = field(default='.')
     """The root path for the source code in the repository.
 
     Example:
         If the `code.filename` is `/path/to/project/src/logfire/main.py` and the `root_path` is `src/`, the URL
         for the source code will be `src/path/to/project/src/logfire/main.py`.
     """
 
+    work_dir: str = field(default='.')
```

Comment:
Do we need this setting at all? Isn't it just `os.getcwd()`?

Reply:
Oh, I see what you mean...

Reply:
True.